libexec/pool/pool-update

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+### pool-update -- update pool metadata
+## Usage: pool update <root>
+
+[[ $(pool check-registry $1) == "up to date" ]] && echo "up to date" && exit
+
+LIVE=true
+POOL_METADATA="$BASE_DIR/pools/$NETWORK/metadata/$1.json"
+POOL_SLUG="\"$(cat $POOL_METADATA | jq -r ".slug")\""
+METADATA_HASH="\"$(pool pin-file $POOL_METADATA)\""
+
+seth send $(pool --registry-address) 'file(address, bool, string, string)' $1 $LIVE $POOL_SLUG $METADATA_HASH
```

Comment:
Maybe this should accept a second parameter `LIVE`, so we can archive pools using the CLI as well?

Reply:
I think we should add this to the pool specific json and then read it from there but I wanted to first hear what you think.

Reply:
Ah then it's persisted, that definitely makes more sense!